### ISSUE TITLE
Is iHD driver thread-safe?

### ISSUE THREAD

**Issue by daijh** [intel/media-driver#1093]
A recent libva PR imposed the requirement that all backends must be thread-safe.
Does iHD driver implement comply with this libva thread-safe statement?

> All VAAPI functions implemented in libva are thread-safe. For any VAAPI
> function that requires the implementation of a backend (e.g. hardware driver),
> the backend must ensure that its implementation is also thread-safe. If the
> backend implementation of a VAAPI function is not thread-safe then this should
> be considered as a bug against the backend implementation. 

**Update va.h for multi-threaded usages** https://github.com/intel/libva/pull/347

Thanks for the clarification.

**Comment by XinfengZhang** [comment]
yes, iHD implementation comply with the libva thread-safe statement.
it was tested by https://github.com/Intel-Media-SDK/samples/tree/master/samples/sample_multi_transcode. 
if there are thread safe issue, please report ,it, maintainers will fix it. 
please also pay attention on these statement of libva:
 When multiple threads are operating on
 the same VAAPI objects, it is the application's responsibility to synchronize
 these operations in order to generate the expected results. For example, using
 a single VAContext from multiple threads may generate unexpected results.
